internal/pkg/vm/actor/builtin/account/account.go

```diff
@@ -39,14 +53,58 @@ func UpgradeActor(a *actor.Actor) error {
 var _ dispatch.ExecutableActor = (*Actor)(nil)
 
 // signatures are the publicly (externally callable) methods of the AccountActor.
-var signatures = dispatch.Exports{}
+var signatures = dispatch.Exports{
+	Constructor: &dispatch.FunctionSignature{
+		Params: []abi.Type{abi.Address},
+		Return: []abi.Type{},
+	},
+}
 
 // Method returns method definition for a given method id.
-func (*Actor) Method(id types.MethodID) (dispatch.Method, *dispatch.FunctionSignature, bool) {
+func (a *Actor) Method(id types.MethodID) (dispatch.Method, *dispatch.FunctionSignature, bool) {
+	switch id {
+	case Constructor:
+		return reflect.ValueOf((*Impl)(a).Constructor), signatures[Constructor], true
+	}
 	return nil, nil, false
 }
 
 // InitializeState for account actors does nothing.
-func (*Actor) InitializeState(_ runtime.Storage, _ interface{}) error {
-	return nil
+func (*Actor) InitializeState(storage runtime.Storage,  initializerData interface{}) error {
+	state, ok := initializerData.(*State)
+	if !ok {
+		return errors.NewFaultError("Initial state to account actor is not a account.State struct")
+	}
+
+	if state.Address.Protocol() != address.SECP256K1 && state.Address.Protocol() != address.BLS {
+		return errors.NewRevertError("Attempt to create account actor with wrong type of address")
+	}
+
+	stateBytes, err := encoding.Encode(state)
+	if err != nil {
+		return xerrors.Wrap(err, "failed to cbor marshal objecinitalizerDatat")
+	}
+
+	id, err := storage.Put(stateBytes)
+	if err != nil {
+		return err
+	}
+
+	return storage.LegacyCommit(id, cid.Undef)
+}
+
+//
+// vm methods for actor
+//
+
+// Impl is the VM implementation of the actor.
+type Impl Actor
+
+// Constructor initializes the actor's state
+func (impl *Impl) Constructor(ctx runtime.InvocationContext, addr address.Address) (uint8, error) {
```

Comment:
Having trouble finding where this gets called

Reply:
This gets called by `vmcontext.CreateActor`. It might not have been wired that way when you made this comment.